File: app/src/main/java/br/com/testeandroid/model/Screen.java
package br.com.testeandroid.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class Screen {

    @SerializedName("title")
    private String title;
    @SerializedName("fundName")
    private String fundName;
    @SerializedName("whatIs")
    private String whatIs;
    @SerializedName("definition")
    private String definition;
    @SerializedName("riskTitle")
    private String riskTitle;
    @SerializedName("risk")
    private int risk;
    @SerializedName("infoTitle")
    private String infoTitle;
    @SerializedName("moreInfo")
    private MoreInfo moreInfo;
    @SerializedName("info")
    private ArrayList<Info> info;
    @SerializedName("downInfo")
    private ArrayList<DownInfo> downInfo;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getFundName() {
        return fundName;
    }

    public void setFundName(String fundName) {
        this.fundName = fundName;
    }

    public String getWhatIs() {
        return whatIs;
    }

    public void setWhatIs(String whatIs) {
        this.whatIs = whatIs;
    }

    public String getDefinition() {
        return definition;
    }

    public void setDefinition(String definition) {
        this.definition = definition;
    }

    public String getRiskTitle() {
        return riskTitle;
    }

    public void setRiskTitle(String riskTitle) {
        this.riskTitle = riskTitle;
    }

    public int getRisk() {
        return risk;
    }

    public void setRisk(int risk) {
        this.risk = risk;
    }

    public String getInfoTitle() {
        return infoTitle;
    }

    public void setInfoTitle(String infoTitle) {
        this.infoTitle = infoTitle;
    }

    public MoreInfo getMoreInfo() {
        return moreInfo;
    }

    public void setMoreInfo(MoreInfo moreInfo) {
        this.moreInfo = moreInfo;
    }

    public ArrayList<Info> getInfo() {
        return info;
    }

    public void setInfo(ArrayList<Info> info) {
        this.info = info;
    }

    public ArrayList<DownInfo> getDownInfo() {
        return downInfo;
    }

    public void setDownInfo(ArrayList<DownInfo> downInfo) {
        this.downInfo = downInfo;
    }
}

File: app/src/main/java/andcordeiro/com/br/testeandroid/entities/Screen.kt
package andcordeiro.com.br.testeandroid.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Screen:Serializable {

    @SerializedName("title")
    var title: String? = null
    @SerializedName("fundName")
    var fundName: String? = null
    @SerializedName("whatIs")
    var whatIs: String? = null
    @SerializedName("definition")
    var definition: String? = null
    @SerializedName("riskTitle")
    var riskTitle: String? = null
    @SerializedName("risk")
    var risk: Int? = null
    @SerializedName("infoTitle")
    var infoTitle: String? = null
    @SerializedName("moreInfo")
    var moreInfo: MoreInfo? = null
    @SerializedName("info")
    var info: List<Info>? = null
    @SerializedName("downInfo")
    var downInfo: List<DownInfo>? = null

    override fun toString(): String {
        return "Screen(title=$title, fundName=$fundName, whatIs=$whatIs, " +
                "definition=$definition, riskTitle=$riskTitle, risk=$risk, infoTitle=$infoTitle, " +
                "moreInfo=$moreInfo, info=$info, downInfo=$downInfo)"
    }
}